### ISSUE TITLE
Is everything seeded?

### ISSUE THREAD

**Issue by daffidwilde** [nicodv/kmodes#64]
Whilst adding in some initialisation process I have been toying with, I've noticed that my results (and the results found with other initialisation methods) are not 100% reproducible. In scikit-learn's implementations, all classes, methods, etc. carry a `random_state` or seed for their random number generators so that results can be reproduced. This could make all the difference here since `np.argsort` (for instance) breaks ties at random. 

Could this feature be added so as to enhance the package, and fall in line with the `sklearn` standard?

**Comment by nicodv** [comment]
If you look in the tests, I've been using `np.random.seed()` for that purpose.
